ActiveQuery.php

```diff
@@ -369,6 +365,19 @@ protected function executeScript($db, $type, $columnName = null)
      */
     private function findByPk($db, $type, $columnName = null)
     {
+        if (!empty($this->orderBy) and in_array($type, ['All', 'One', 'Count', 'Column'])) {
+            $orderBy = array_slice($this->orderBy, 0);
```

Comment:
Isn't `array_slice($this->orderBy, 0)` gives us a copy of the same array? Can't it be simple `=`?